lib/devices/ios/ios.js

```diff
@@ -429,6 +429,17 @@ IOS.prototype.setInitialOrientation = function (cb) {
   }
 };
 
+IOS.isSpringBoard = function (uiAppObj) {
+  var isSpringBoard = false;
+  var metaData = uiAppObj['@'];
+
+  if (_.isObject(metaData) && _.has(metaData, "name")) {
```

Comment:
actually, I think this whole function can just be:

```js
IOS.isSpringBoard = function (uiAppObj) {
  return _.propertyOf(uiAppObj['@'])('name') === 'SpringBoard';
}
```

Up to you if you still want to make it its own function. It is nice for the unit test that way.